packages/experiments/experimental-web-controls/src/Link/Link.settings.ts

```diff
@@ -5,8 +5,8 @@ export const settings: IComposeSettings<ILinkType> = [
   {
     tokens: {
       fontFamily: 'primary',
-      fontSize: 'medium',
-      fontWeight: 'medium',
+      fontSize: 'secondary',
+      fontWeight: 'regular',
```

Comment:
This may have been changed by mistake - do I want to edit these values in experimental-web-controls?

Reply:
I don't think so, I think experimental web-controls will probably be removed at some point.  I'd like to get react-native-web running, and have the tester be able to render there, at that point there will be no point to the experimental web controls.

If they are becoming a maintenance nightmare maybe we should remove them earlier.